docs/dev-guide/installingNative.md

```diff
@@ -157,6 +157,7 @@ Most of these prerequisites can be installed using the package manager of your O
 
   ```sh
   make deps
+  make e2e-deps
```

Comment:
If we add `e2e-deps` as a dependency for the `test-e2e` target, I believe we can remove the mentions in this file.